include/evmc/evmc.h

```diff
@@ -432,6 +432,52 @@ struct evmc_result
 typedef bool (*evmc_account_exists_fn)(struct evmc_host_context* context,
                                        const evmc_address* address);
 
+/**
+ * Access status per EIP-2929: Gas cost increases for state access opcodes.
+ */
+enum evmc_access_status
+{
+    /**
+     * The entry hasn't been accessed before – it's the first access.
+     */
+    EVMC_ACCESS_COLD = 0,
+
+    /**
+     * The entry is already in accessed_addresses or accessed_storage_keys.
+     */
+    EVMC_ACCESS_WARM = 1
+};
+
+/**
+ * Access account callback function (EIP-2929).
+ *
+ * This callback function is used by a VM to add the given address
+ * to accessed_addresses substate (EIP-2929).
+ *
+ * @param context  The pointer to the Host execution context.
```

Comment:
This other places use this:
```suggestion
 * @param context  The Host execution context.
```